src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml

```diff
@@ -0,0 +1,144 @@
+    var showJobRoleError = !String.IsNullOrEmpty(jobRoleError);
+
+    var timeInRoleError = showErrors && Model.ErrorMessages.FirstOrDefault(x => x.Field == "TimeInRole") != null ? "govuk-input--error" : "";
+    var showTimeInRoleError = !String.IsNullOrEmpty(timeInRoleError);
+    var isPartOfOtherOrgError = showErrors && Model.ErrorMessages.FirstOrDefault(x => x.Field == "IsPartOfOtherOrgThatGetsFunding") != null ? "govuk-input--error" : "";
+    var showIsPartOfOtherOrgError = !String.IsNullOrEmpty(isPartOfOtherOrgError);
+ 
+    var errorMessages = Model.ErrorMessages != null ? Model.ErrorMessages.DistinctBy(x => x.ErrorMessage) : null;
+
+    var checkedOtherOrgYes = Model.IsPartOfOtherOrgThatGetsFunding=="Yes" ? "checked": null;
+    var checkedOtherOrgNo = Model.IsPartOfOtherOrgThatGetsFunding =="No" ? "checked": null;
+}
+
+<div class="govuk-form-group @(showNameError ? "govuk-form-group--error" : "")" sfa-validation-for="FullName" sfa-validationerror-class="form-group-error">
+
+    <label for="FullName" class="govuk-label">Full name</label>
+    @if (showErrors)
+    {
+        <span class="govuk-error-message">
+            @foreach(var nameErrorMessage in errorMessages.Where(x => x.Field == "FullName"))
+            {
+                @Html.Raw(nameErrorMessage.ErrorMessage) <br />
+            }
+        </span>
+    }
+    <input id="FullName" name="FullName" type="text" class="govuk-input @nameError" asp-for="FullName"
+           sfa-validation-for="PersonName" sfa-validationerror-class="form-control-error" />
+</div>
+<div class="govuk-form-group @(showJobRoleError ? "govuk-form-group--error" : "")" sfa-validation-for="JobRole" sfa-validationerror-class="form-group-error">
+
+    <label for="JobRole" class="govuk-label">Job Role</label>
+    @if (showErrors)
+    {
+    <span class="govuk-error-message">
+        @foreach(var jobRoleErrorMessage in errorMessages.Where(x => x.Field == "JobRole"))
+            {
+        @Html.Raw(jobRoleErrorMessage.ErrorMessage)  <br />
+            }
+    </span>
+    }
+    <input id="JobRole" name="JobRole" type="text" class="govuk-input @jobRoleError" asp-for="JobRole"
+           sfa-validation-for="PersonJobRole" sfa-validationerror-class="form-control-error" />
+</div>
+
+<div class="govuk-form-group @((showTimeInRoleError) ? "govuk-form-group--error" : "")" sfa-validation-for="TimeInRoleMonths" sfa-validationerror-class="form-group-error">
+
+    <p class="govuk-label">Time in role</p>
+    <span class="govuk-hint" id="dob-hint">
+        For example, 3 02
+    </span>
+    @if (showErrors)
+    {
+        <span class="govuk-error-message">
+            @foreach (var timeInRoleMonthsMessage in errorMessages.Where(x => x.Field == "TimeInRole"))
+            {
+                @Html.Raw(timeInRoleMonthsMessage.ErrorMessage) <br />
+            }
+        </span>
+    }
+    <div class="govuk-date-input" id="timeInRole">
```

Comment:
Potential for re-use of this year/months control, perhaps it could be added to the QnA Api Views package?